packages/web-react/src/components/Matrix/demo/MatrixResponsiveColumnsGap.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import { ScrollView } from '../../ScrollView';
+import Matrix from '../Matrix';
+import MatrixItemFactory from './MatrixItemFactory';
+
+const columns = [
+  [
+    'Stack 1',
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec a diam lectus. Sed sit amet ipsum mauris.',
+    'Lorem ipsum',
+  ],
+  [
+    'Stack 2',
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec a diam lectus. Sed sit amet ipsum mauris.',
+  ],
+  [
+    'Stack 3',
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec a diam lectus. Sed sit amet ipsum mauris.',
+    'Lorem ipsum',
+  ],
+  [
+    'Stack 4',
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit.',
+    'Lorem ipsum dolor sit amet, consectetur adipiscing elit. Donec a diam lectus. Sed sit amet ipsum mauris.',
+  ],
+];
+
+const MatrixResponsiveColumnsGap = () => (
+  <ScrollView direction="horizontal" data-spirit-toggle="scrollView">
+    <Matrix
+      cols={{ mobile: 1, tablet: 2 }}
+      itemRows={3}
+      rows={{ mobile: 4, tablet: 2 }}
```

Comment:
Strangely enough, `Matrix` seems to work even when `rows` are random numbers (`itemRows` is what must be set correctly). I can investigate it later, but for now, please update the example so it uses numbers that equal to `<NUMBER_OF_ITEMS> * itemRows / cols`:

```suggestion
      rows={{ mobile: 12, tablet: 6 }}
```

Reply:
Maybe `rows` (if not set) can be calculated based on `cols`, `itemRows`, and `Children.count(children)` in React? 🤔 It would make developer's life easier.